ecwid_api.gemspec

```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 0"
   spec.add_development_dependency "rspec", "~> 2.14", ">= 2.14.1"
 
-  spec.add_dependency "faraday", "~> 0.9.0"
+  spec.add_dependency "faraday", ">= 0.9.0"
```

Comment:
might be unsafe..if we test it with the version we'd like to have then can update it based on that. E.x if we know it works with 0.10 we can update to "~> 0.10" instead. Otherwise if it could breaks on faraday version 1.2 or something since we never tested that and this would allow that upgrade to happen

Reply:
I can update it to `~> 0.10.0` so we'll be locked to >= 0.10 and < 0.11

Reply:
@slamorsi updated.